client/json-rpc/rpc_procedures.c

```diff
@@ -496,6 +500,60 @@ cJSON * method_xdag_get_transactions(struct xdag_rpc_context * ctx, cJSON * para
 	return result;
 }
 
+/* create new addresss */
+/*
+ request:
+ "method":"xdag_new_address", "params":[{"number":1}], "id":1
```

Comment:
Could you please clarify params and result.

Reply:
Number is the number of address to be generated.

Result is the array of generated addresses.

Reply:
Do you mean count of addresses?

Reply:
Good idea